transcriptic/cli.py

```diff
@@ -628,3 +628,66 @@ def login_cmd(ctx, api_root=None, analytics=True, rsa_key=None):
 def format_cmd(manifest):
     """Check Autoprotocol format of manifest.json."""
     commands.format(manifest)
+
+
+@cli.command("exec")
+@click.argument("autoprotocol", type=click.File("r"), default=sys.stdin)
+@click.option(
+    "--api", "-a", help="The api endpoint of your scle test workcell instance."
+)
+@click.option(
+    "--workcell_id",
```

Comment:
my bad. I meant kebab-case for options... Should be a straightforward change though